auth_otp/admin.py

```
@@ -1,3 +1,242 @@
+        'otp_code',
+        'expires_at',
+        'created_at',
+        'ip_address',
+        'user_agent',
+        'kavenegar_message_id',
+        'metadata'
+    ]
+    
+    ordering = ['-created_at']
+    
+    def otp_code_display(self, obj):
```

Ruff (0.12.2)

Missing return type annotation for public function `otp_code_display`

(ANN201)